tests/test_external_search.py

```diff
@@ -54,259 +62,225 @@ def teardown(self):
 
 
 class TestExternalSearch(ExternalSearchTest):
-    """
-    These tests require elasticsearch to be running locally. If it's not, or there's
-    an error creating the index, the tests will pass without doing anything.
 
-    Tests for elasticsearch are useful for ensuring that we haven't accidentally broken
-    a type of search by changing analyzers or queries, but search needs to be tested manually
-    to ensure that it works well overall, with a realistic index.
+    def test_setup_index_creates_new_index(self):
```

Comment:
I didn't change these tests, but I moved them to a different class than the test that needs to create and index lots of works so they don't need to run the slow setup method.